Sources/tuist/tuist.docc/Articles/Contributing/Contributing - Get Started.md

```diff
@@ -0,0 +1,54 @@
+#  Get Started
+
+Tuist is a command line interface (CLI), written in Swift, that helps developers maintain and interact with their Xcode projects.
+It also abstracts them from the complexities that Xcode exposes.
+
+If you have experience building apps for Apple platforms, like iOS, adding code to Tuist shouldn't be much different. There are two differences compared to developing apps that are worth mentioning:
+
+- **The interactions with CLIs happen through the terminal.**
+  The user executes Tuist, which performs the desired task, and then returns successfully or with an error code.
+  During the execution, the user can be notified by sending output information to the standard output and standard error.
+  There are no gestures, or graphical interactions, just the user intent.
+- **There's no runloop** that keeps the process alive waiting for input, like it happens in an iOS app when the app receives system or user events.
+  CLIs run in its process and finishes when the work is done.
+  Asynchronous work can be done using system APIs like `DispatchQueue`, but need to make sure the process is running while the asynchronous work is being executed.
+  Otherwise, the process will terminate the asynchronous work.
+
+If you don't have any experience with Swift, we recommend [Apple's official book](https://docs.swift.org/swift-book/) to get familiar with the language and the most used elements from the Foundation's API.
+
+## Minimum requirements
+
+To contribute to Tuist, minimum requirements are:
+
+- `macOS 14.0+`
+- `Xcode 15.0+`
+
+## Set up the project locally
+
+To start working on the project, we can follow the steps below:
+
+- Clone the repository by running: `git clone git@github.com:tuist/tuist.git`
+- Run `make tuist/generate` to generate the Tuist Xcode project using tuist itself
```

Comment:
We should mention the makefiles, but generally, I think people should use the latest published `tuist` version instead. I'm adding `asdf` in this [PR](https://github.com/tuist/tuist/pull/5553) which should be the primary way people work on tuist. I can update these docs after since this is just migration from what we have right now.

Reply:
Yep, let's do it in a separate PR!